topics/client-responses.md

```diff
@@ -76,8 +76,11 @@ below shows how to get a response as a `ByteArray` and save it to a file:
 The [`onDownload()`](https://api.ktor.io/ktor-client/ktor-client-core/io.ktor.client.plugins/on-download.html) extension
 function in the example above is used to display download progress.
 
-This approach loads the entire response into memory at once, which can be problematic for large files. To reduce memory
-usage, consider [streaming the data](#streaming) in chunks.
+For non-streaming requests, calling `.body()` loads and caches the full response body in memory, allowing repeated 
```

Comment:
It is cached when we call `.get(url)` or other methods with body. And `.body()` just returns the saved response.